app/src/ui/organism/product-fresh/ProductFresh.js

```diff
@@ -0,0 +1,28 @@
+import SectionTitle from "../../molecules/section-title/SectionTitle";
+import BaseProductItem from "../../molecules/base-product-item/BaseProductItem";
+
+const ProductFresh = () => {
```

Comment:
Beautiful component :) one this to fix - if we don't have business logic (no JS in this function/component body) we can return shorthand html 

const ProductFresh = () => (
    YOUR HTML
}

Reply:
What mean  business logic ?